x-pack/plugins/index_management/__jest__/client_integration/home/data_streams_tab.test.ts

```diff
@@ -238,7 +238,7 @@ describe('Data Streams tab', () => {
       const { table, actions } = testBed;
       await actions.clickIndicesAt(0);
       expect(table.getMetaData('indexTable').tableCellsValues).toEqual([
-        ['', '', '', '', '', '', '', 'dataStream1'],
+        ['', 'data-stream-index', '', '', '', '', '', '', 'dataStream1'],
```

Comment:
This is the correct behaviour. Previously `getMetaData()` was only retrieving `td` cell value and omitting `th` cells. The data stream index cell is indicated as `th` and is now returned.

Reply:
Thanks for explaining and for catching this!

Reply:
No prob! 👍